src/classes/PicklistEntryMapper_TEST.cls

```diff
@@ -0,0 +1,95 @@
+    @isTest
+    private static void getInstanceNew() {
+
+        Test.startTest();
+            PicklistEntryMapper picklistEntryMapperInstance = PicklistEntryMapper.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(PicklistEntryMapper.instance, picklistEntryMapperInstance, 'Instance of mapper class returned should match static instance variable.');
+        System.assertEquals(true, picklistEntryMapperInstance != null, 'Instance of mapper class should not be null.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getInstance method returns the existing instance of the 
+    * PicklistEntryMapper class when one already exists.
+    ***************************************************************************************************************************/
+    @isTest
+    private static void getInstanceExisting() {
+
+        Test.startTest();
+            PicklistEntryMapper picklistEntryMapperInstance1 = PicklistEntryMapper.getInstance();
+            PicklistEntryMapper picklistEntryMapperInstance2 = PicklistEntryMapper.getInstance();
+        Test.stopTest();
+
+        System.assertEquals(PicklistEntryMapper.instance, picklistEntryMapperInstance1, 'Instance of mapper class returned should match static instance variable.');
+        System.assertEquals(picklistEntryMapperInstance1, picklistEntryMapperInstance2, 'Subsequent retrievals of mapper class instance should return existing instance.');
+        System.assertEquals(true, picklistEntryMapperInstance1 != null, 'Instance of mapper class should not be null.');
+    }
+
+    /**************************************************************************************************************************
+    * @description Test method to verify that the getModelsByNameForSObjectField method returns the picklist entry models
+    ***************************************************************************************************************************/
+    @isTest
+    private static void getModelsByNameForSObjectField() {
+
+        String prefix = UTIL_Namespace.StrTokenNSPrefix('');
+        String objectName = prefix + 'Relationship__c';
+        String fieldName = prefix + 'Type__c';
+
+        Test.startTest();
+            Map<String, PicklistEntryModel> picklistEntryModelByPicklistEntryName = 
+                PicklistEntryMapper.getInstance().getModelsByNameForSObjectField(UTIL_Describe.getObjectDescribe(objectName).getSObjectType(), 
+                                                                                 UTIL_Describe.getFieldDescribe(objectName, fieldName).getSObjectField());
+        Test.stopTest();
+
+        List<Schema.PicklistEntry> picklistEntriesList = UTIL_Describe.getFieldPicklistEntries(prefix + 'Relationship__c', prefix + 'Type__c');
```

Comment:
Why not use the local variables you defined above for this? That way it will stay consistent.
```suggestion
        List<Schema.PicklistEntry> picklistEntriesList = UTIL_Describe.getFieldPicklistEntries(objectName,fieldName);
```